docs/atomic.md

```diff
@@ -0,0 +1,18 @@
+Atomic host bootstrap
+=====================
+
+Atomic host testing has been done with the network plugin flannel. Change the inventory var `kube_network_plugin: flannel`.
+
+### Vagrant
+
+* For bootstrapping with Vagrant, use box centos/atomic-host 
```

Comment:
Some users use vagrant-libvirt. Does centos/atomic-host have qcow2 format for kvm machines?